Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/safes.yml

```diff
@@ -1,55 +1,52 @@
 - type: entity
-  name: random shotgun safe spawner
-  id: ShotgunSafeSpawner
+  id: BaseGunSafeSpawner
   parent: MarkerBase
+  abstract: true
   components:
     - type: Sprite
       layers:
         - state: red
         - sprite: Structures/Storage/closet.rsi
           state: shotguncase
+    - type: RandomSpawner
+      offset: 0
+
+- type: entity
+  name: random safe spawner
```

Comment:
pro put name on the abstract one

Reply:
no that's the shotgun safe spawner not the abstract one

Reply:
ohh nvm i think i might be dumb sometimes or most of the times